lib/ut/coroutine2.c

```diff
@@ -0,0 +1,266 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2017-2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Update years.

Reply:
Once https://github.com/Seagate/cortx-motr/pull/752 gets merged, I'll rebase my branch and this problem will be resolved. Let's keep this comment open until it happens.